learning_assistant/urls.py

```diff
@@ -24,4 +29,9 @@
         LearningAssistantMessageHistoryView.as_view(),
         name='message-history',
     ),
+    re_path(
+        fr'learning_assistant/v1/course_id/{COURSE_ID_PATTERN}/summary',
+        LearningAssistantSummaryView.as_view(),
+        name='summary',
+    ),
```

Comment:
I think we have plans to enable (and integrate) the Unit Summary to Xpert. Maybe this `/summary` reference could be confusing. Maybe we could use `/chat` instead?

Reply:
To clarify, are you suggesting that I change the URL/name to `chat`? I think that would conflict with the chat endpoint, so I would prefer not to do that. I also don't think `chat` represents the purpose of this endpoint.

Personally, I think `summary` and `unit_summary` would be reasonable URLs to coexist, and I wouldn't find them confusing. I don't think we should name the URL for unit summaries `summary`.

What about one of these? `overview`, `details`, or `info`?

Reply:
I totally forgot about the existing `/chat` one. I think `info` fits better, but it's your call.